inginious/agent/docker_agent/__init__.py

```diff
@@ -188,14 +189,20 @@ def environments(self):
                 return
 
             task_path = os.path.join(container_path, 'task')  # tmp_dir/id/task/
+            course_path = os.path.join(container_path, 'course')
+
             sockets_path = os.path.join(container_path, 'sockets')  # tmp_dir/id/socket/
             student_path = os.path.join(task_path, 'student')  # tmp_dir/id/task/student/
             systemfiles_path = os.path.join(task_path, 'systemfiles')  # tmp_dir/id/task/systemfiles/
 
+            course_common_path = os.path.join(course_path, 'common')
```

Comment:
Non-tasks folders should not be confused with tasks folder. I suggest to use the same naming a for translations files, ``$common`` and ``$common_student``.